spec/lib/models/git_repository_spec.rb

```diff
@@ -77,14 +77,14 @@
   describe '#push' do
     it 'calls run_command with correct parameters' do
       allow(subject).to receive(:within_repo).and_yield
-      expect(subject).to receive(:run_command).with("git push")
+      expect(subject).to receive(:run_command).with('git push')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.